src/map/clif.c

```diff
@@ -17527,6 +17536,26 @@ void clif_party_leaderchanged(struct map_session_data *sd, int prev_leader_aid,
 }
 
 /**
+ * Decrypt packet identifier for player
+ * @param fd
+ * @param sd
+ * @param packet_ver
+ * Orig author [Ind/Hercules]
+ **/
+static unsigned short clif_parse_cmd(int fd, struct map_session_data *sd) {
+#ifndef PACKET_OBFUSCATION
+	return RFIFOW(fd, 0);
+#else
+	unsigned short cmd = RFIFOW(fd,0);
+	if (sd)
+		cmd = (cmd ^ ((sd->cryptKey >> 16) & 0x7FFF));
+	else
```

Comment:
// A player tries to connect - use the initial keys for the decryption of the packet ID